### ISSUE TITLE
[DataCap Application] <LZ> 

### ISSUE THREAD

**Issue by Tom88881025** [Marshall-btc/Marshall-Fil-Data-Pathway#1]
### Data Owner Name

Li Zhu Pharmaceutical Group Co.

### Data Owner Country/Region

China

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.livzon.com.cn/intro/2.html

### Social Media Handle

https://www.livzon.com.cn/intro/2.html

### Social Media Type

Other

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

580TiB

### Number of replicas to store

5

### Weekly allocation of DataCap requested

1000TiB

### On-chain address for first allocation

f1hkqml3xxghwz5h3q4og2convvnelr5wcoy6pgcy

### Data Type of Application

Public, Open Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Founded in January 1985, with a registered capital of 935 million yuan, Lizhu Pharmaceutical Group Co., Ltd. is a comprehensive pharmaceutical group company integrating pharmaceutical research and development, production and sales, with more than 8,000 employees, and a listed company on A+H shares.In FY2022, the company's operating revenue was 12.63 billion yuan, with a net profit of 1.909 billion yuan, and an investment of 1.401 billion yuan in research and development.
 
Lizhu has been making breakthroughs in the fields of APIs, chemical drugs, traditional Chinese medicines, biological drugs, in vitro diagnostic reagents, etc. Its products are distributed in the fields of digestion, cardiovascular and cerebral vascular, reproductive endocrine, and psychiatric/neurological. The company continues to focus on the main business of innovative drugs and high-barrier complex formulations, launching heavyweight products such as leucovorin and eplazole, and realizing volume growth in the fields of digestion and reproduction.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Medical experiment data, medical education data
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

None

### Please share a sample of the data

```text
https://www.livzon.com.cn/intro/2.html
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, Africa, North America, South America, Europe, Australia (continent), Antarctica

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
SPs in communication:
f03074583  
f03074586   
f03074587   
f03074589   
f03074592
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Droplet client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Tom88881025** [comment]
KYC mail has been sent, please pay attention to review, thank you!

<img width="1227" alt="image" src="https://github.com/Marshall-btc/Marshall-Fil-Data-Pathway/assets/116630682/b9dec0bf-c933-4fb6-bda5-f418322ca91d">

**Comment by Marshall-btc** [comment]
![image](https://github.com/Marshall-btc/Marshall-Fil-Data-Pathway/assets/70891974/a6bc7be5-1c9c-4e36-8ed2-1f2fb7e31901)
I have received a KYC email from this client.

Before the first round begins, I still want to remind you @Tom88881025  :

You and SP are to strictly adhere to the rules of Fil+ and Marshall-Fil-Data-Pathway.
Please confirm SP's geographic location and name before the start of each allocation round. If there is a change, please comment promptly in that application.
Please keep the SP retrievable.

**Comment by Tom88881025** [comment]
Yes, we'll follow the rules.The first round of storage will start at the end of June/beginning of July, and more detailed information on SPs will be synchronized then.

**Comment by Tom88881025** [comment]
checker:manualTrigger

**Comment by Tom88881025** [comment]
Updating the sealing data information before the next round is about to start is currently in continuous communication with SPs such as f03091739, f03144037, f01106668 , f01315096, f0870558, f03055005, f03055018, f03055029 and others.

**Comment by Marshall-btc** [comment]
checker:manualTrigger

**Comment by Marshall-btc** [comment]
Hey! @Tom88881025 I see that you've completed the first round of encapsulation and have two questions I'd like you to answer.
1. Why did the SP not appear in your application for this round, could you please explain?
2. Why the retrieval success rate of these SPs is very low?

**Comment by Marshall-btc** [comment]
By the way, from the checking result, there is no CID sharing and the SPs are in different cities and countries.

**Comment by Tom88881025** [comment]
> ![image](https://private-user-images.githubusercontent.com/70891974/349398203-f2345e11-1b7a-4b1c-b714-a71b950fe69f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyMDMxNzMsIm5iZiI6MTcyMTIwMjg3MywicGF0aCI6Ii83MDg5MTk3NC8zNDkzOTgyMDMtZjIzNDVlMTEtMWI3YS00YjFjLWI3MTQtYTcxYjk1MGZlNjlmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE3VDA3NTQzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxZTkxOGU3N2QzYzk5YWFhYjlmYjc5YTFjZGM1ZjJmNGZmYzExNmE4ZjI2ODYyMmJhOGMzZDRjMmU1NzIyNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vF_8J6MPJwVF06wFOiWisd1JSXSKhmfj59vAVEUe7QA)
> 
> Hey! @Tom88881025 I see that you've completed the first round of encapsulation and have two questions I'd like you to answer.
> 
> 1. Why did the SP not appear in your application for this round, could you please explain?
> 2. Why the retrieval success rate of these SPs is very low?

Thank you for your review and I will respond as follows:
1. fil-plus data sealing requires data cut preparation, Datacap quota, filecoin pledge coins to be satisfied at the same time, the timing is not very compatible with the SPs that were applied for disclosure at that time, so we re-found new SPs that meet the timing.
2. I have communicated with SPs, they told me that the fil-plus retrieval method is now changed to a new mode Spark, the team is researching and troubleshooting, Spark retrieval success rate is low for a variety of reasons, it may be the SPs network problems, or it may be that the Spark retrieval did not retrieve. Please continue to pay attention to the follow-up Spark retrieval success rate.
We will be more and more compliant to carry out.

**Comment by Marshall-btc** [comment]
I will approve this round, but also expect you to follow the FIL+ rules. Please add more SPs in the next round and also improve the retrieval success rate. If things don't change in the next round then I will consider suspending your Datacap allocation.

**Comment by Tom88881025** [comment]
checker:manualTrigger

**Comment by Tom88881025** [comment]
@Marshall-btc  I spoke with SPs again, and we always knew that the community consensus search methods were GraphSync, Bitswap, and Http, and for that reason we had been using the results of those three search methods as the markers of a successful search until we learned at the notarization meeting that the newest search criteria were based on the results on https://spacemeridian.grafana.net /public-dashboards/32c03ae0d89748e3b08e0f08121caa14?orgId=1 on the results.

I asked SP to study the Spark retrieval principle and adjust the configuration to fit the Spark retrieval requirements.SP feedback felt that the code of spark is different from the code of other retrieval methods used before, and it takes time to study and analyze. Please support it.

**Comment by Marshall-btc** [comment]
checker:manualTrigger

**Comment by Marshall-btc** [comment]
![image](https://github.com/user-attachments/assets/ed3a1167-d4df-49cc-83f2-9e0f06dc6db8)
Hey @Tom88881025 I see that the retrieval success rate of f03144037 is improving, but the retrieval success rate of f03055005, f01315096, and f03091739 is still very low, can you explain some why?

**Comment by Tom88881025** [comment]
Yes, there are time differences in each SPs understanding of Spark retrieval, including differences in each SPs team's technical level of analyzing Spark, and some of the larger SPs in the community have also provided feedback in the community in the hope that the Spark team can disclose the entire principle. We believe that the Spark team is also working hard to improve. We hope to continue to support, we will continue to work hard.

**Comment by Marshall-btc** [comment]
Ok, I'll support this round, but still want you to contact SPs to improve retrieval success.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by Tom88881025** [comment]
Due to recent changes in staking coins, the list of SPs we are currently cooperating with and in communication with is as follows. We expect to initiate soon:
f01106668、f0870558、f01315096、f03055005、f03055018、f03055029、f01518369、f01889668、f03151449、f03151456、f03179555、f03179570、f03188440、f03190614、f03190616.

**Comment by Marshall-btc** [comment]
checker:manualTrigger

**Comment by Marshall-btc** [comment]
1. Hey, I noticed you updated the list of SPs in the comments with a bunch of them. Can you tell me the reason for doing that?
![image](https://github.com/user-attachments/assets/3182393c-889e-40aa-b1c6-4b079e53513d)

2. One more question. I see that most of the SPs can be retrieved by spark. What is the reason for these SPs not being able to support spark retrieval yet?
![image](https://github.com/user-attachments/assets/fc99b8f3-71ef-4133-a76f-c28dfb5d5fb0)

**Comment by Tom88881025** [comment]
Apologies for the late reply.As you can see, we are currently in communication with the listed SPs, and the latest sealing plan is about to launch. Most SPs have implemented Spark retrieval after our repeated discussions. However, some SPs’ operations teams, due to other projects, have not fully realized Spark retrieval yet. Nevertheless, f03055005, f03055029, and f03055018 have committed to achieving Spark retrieval by the end of the month, with at least 30% completion by mid-October, and they will continue to improve. Please stay tuned.Thank you very much！

**Comment by Marshall-btc** [comment]
Per the last community meeting and the Governance Team's request for Allocator review. I will be reviewing a random sample of SPs in your application. @Tom88881025 
Please send documentation proving location information for f03055005 to me at marshallbtc1990@gmail.com Documentation can be a lease, utility bill, etc.

**Comment by Tom88881025** [comment]
Please check your e-mail，thanks！
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/7ffe3c65-678a-49ff-a93b-562f78ca1cd5">

**Comment by Marshall-btc** [comment]
Hey @Tom88881025  , I've received your email and I'd like to believe that your SP is in the Hong Kong. I can support this round, but hopefully you will continue to contact SPs to improve Spark retrieval success.

**Comment by Marshall-btc** [comment]
checker:manualTrigger

**Comment by Marshall-btc** [comment]
![image](https://github.com/user-attachments/assets/f6ee6b5a-957e-4cb3-80c0-e32fe2092615)
The CID checker looks like everything is improving when comparing the results from last time. I can support this round and I hope you @Tom88881025 can contact these two SPs to improve their spark retrieval success rate.

**Comment by Tom88881025** [comment]
We have already contacted these two SPs, and they have reported that the Spark retrieval rate will significantly improve by the end of the month. Please continue to monitor the situation. Thank you.

**Comment by Marshall-btc** [comment]
![image](https://github.com/user-attachments/assets/e2ee2113-e469-4548-bbc8-f093d6d4e034)
Overall it looks good, based on your Datacap application I will continue to support the remaining amount and hope you perform as well as ever. @Tom88881025

**Comment by Tom88881025** [comment]
Pre-announce that the new SP: f03300963 is about to join.

**Comment by Tom88881025** [comment]
Updation：f01083939 、f01083949.

**Comment by Tom88881025** [comment]
@Marshall-btc Through browser inquiries, it appears that the quota has already been used up, but the robot does not show this. Could you help me check the backend system?

<img width="990" alt="Image" src="https://github.com/user-attachments/assets/8544317c-a44f-4f9d-9a2a-e0a6765ee0c8" />

**Comment by Marshall-btc** [comment]
checker:manualTrigger

**Comment by Marshall-btc** [comment]
I applaud you for updating the list of SPs as always, but am very sorry that the application application quota has been exhausted and I will not be making any further allocations. I will close it manually. But you are still very welcome to introduce new datasets.

**Comment by Marshall-btc** [comment]
checker:manualTrigger

**Comment by Tom88881025** [comment]
checker:manualTrigger

**Comment by Tom88881025** [comment]
The allocator contacted me via Slack and asked me to update the SPs list. It has been completed.
